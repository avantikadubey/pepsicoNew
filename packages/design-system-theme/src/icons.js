import { faGoogle, faAmazon } from '@fortawesome/free-brands-svg-icons';
import {
  faChevronRight,
  faChevronLeft,
  faRupeeSign,
  faDollarSign,
  faEye,
  faBars,
  faSave,
  faUndo,
  faLink,
  faEdit,
  faImage,
  faUpload,
  faFileAlt,
  faFileEdit,
  faFilePlus,
	faTrash,
	faSignOut,
  faTrashAlt,
  faArrowRight,
  faPlayCircle,
  faPlusCircle,
  faCheck,
  faHourglass,
  faSignOutAlt,
  faHourglassStart,
  faHourglassEnd,
  faCheckCircle,
  faCog,
  faPlusSquare,
  faMinusCircle,
  faExclamationCircle,
  faExclamationTriangle,
  faSlidersH,
  faSync,
  faBan,
  faWatch,
  faCubes,
  faShoppingBasket,
  faShoppingCart,
  faAngleLeft,
} from '@fortawesome/pro-regular-svg-icons';
import { faPlus, faMinus, faLockAlt } from '@fortawesome/pro-solid-svg-icons';
import {
  faExternalLinkSquareAlt,
  faCalendarAlt,
  faClock,
  faFileUpload,
  faTimesCircle,
  faTimes,
  faInfo,
  faClipboardList,
  faPenSquare,
	faPlay,
	faUserCircle,
  faCogs,
  faPen,
  faInfoCircle,
  faUser,
  faPencilAlt,
  faArrowLeft,
  faDownload,
  faCaretUp,
  faCaretDown,
  faSearch,
  faCaretRight,
  faCaretLeft,
  faArrowAltCircleLeft,
  faSyncAlt
} from '@fortawesome/free-solid-svg-icons';
import { library } from '@fortawesome/fontawesome-svg-core';

/**
 * all icons needed must be added to the library in order to be used.
 * @see https://github.com/FortAwesome/react-fontawesome#build-a-library-to-reference-icons-throughout-your-app-more-conveniently
 * @see https://fontawesome.com/icons?d=gallery
 */
library.add(
  faGoogle,
  faChevronRight,
  faChevronLeft,
  faRupeeSign,
  faDollarSign,
  faEye,
  faBars,
  faEdit,
  faLockAlt,
  faFileEdit,
  faFilePlus,
  faLink,
  faSearch,
	faUser,
	faUserCircle,
	faSignOut,
  faSignOutAlt,
  faInfo,
  faArrowRight,
  faCheck,
  faHourglassStart,
  faHourglassEnd,
  faExclamationTriangle,
  faSave,
  faUndo,
  faFileAlt,
  faPlay,
  faSync,
  faImage,
  faUpload,
  faCubes,
  faPen,
  faPenSquare,
  faBan,
  faCogs,
  faTrash,
  faTrashAlt,
  faWatch,
  faPlus,
  faClipboardList,
  faExclamationCircle,
  faFileUpload,
  faPlusCircle,
  faPlusSquare,
  faPlayCircle,
  faCheckCircle,
  faMinusCircle,
  faHourglass,
  faSlidersH,
  faCalendarAlt,
  faTimesCircle,
  faTimes,
  faExternalLinkSquareAlt,
  faClock,
  faShoppingBasket,
  faPlus,
  faMinus,
  faAngleLeft,
  faCog,
  faInfoCircle,
  faAmazon,
  faShoppingCart,
  faPencilAlt,
  faArrowLeft,
  faDownload,
  faCaretUp,
  faCaretDown,
  faCaretRight,
  faCaretLeft,
  faArrowAltCircleLeft,
  faSyncAlt
);
