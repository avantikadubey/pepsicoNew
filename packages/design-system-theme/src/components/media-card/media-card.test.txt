import React from 'react'
import renderer from 'react-test-renderer'
import { config } from '@fortawesome/fontawesome-svg-core'
import '@fortawesome/fontawesome-svg-core/styles.css'
import MediaCard from './media-card'
import '../../icons'

beforeAll(() => {
	config.autoAddCss = false
})

test('Media Card Creation', () => {
	const component = renderer.create(<MediaCard />)
	const cardComponent = component.toJSON()
	expect(cardComponent).toMatchSnapshot()
})
