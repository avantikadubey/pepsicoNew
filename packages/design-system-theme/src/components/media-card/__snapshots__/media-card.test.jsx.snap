// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Media Card Creation 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-1 MuiPaper-rounded"
>
  <div
    className="MuiCardMedia-root makeStyles-media-8"
    style={
      Object {
        "backgroundImage": "url(\\"store/planogram-store.jpg\\")",
      }
    }
    title="Contemplative Reptile"
  />
  <div
    className="MuiCardContent-root"
  >
    <div
      className="MuiGrid-root makeStyles-description-2 MuiGrid-container"
    >
      <div
        className="MuiGrid-root makeStyles-salesAllocation-3 MuiGrid-item MuiGrid-grid-xs-12"
      >
        <h3
          className="MuiTypography-root MuiTypography-h3"
        >
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textField-9"
          >
            <div
              className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
              onClick={[Function]}
            >
              <input
                aria-invalid={false}
                className="MuiInputBase-input MuiInput-input"
                disabled={false}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                type="text"
                value={0}
              />
            </div>
          </div>
          %
        </h3>
        <span
          className="MuiTypography-root"
        />
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
      >
        <h5
          className="MuiTypography-root makeStyles-planogramName-7 MuiTypography-h5 MuiTypography-colorTextSecondary"
        >
          XXX
        </h5>
      </div>
    </div>
  </div>
  <div
    className="MuiCardActions-root makeStyles-edit-4 MuiCardActions-spacing"
  >
    <button
      className="MuiButtonBase-root MuiButton-root makeStyles-editIcon-5 MuiButton-text"
      disabled={false}
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-pencil-alt fa-w-16 fa-2x makeStyles-icon-6"
          data-icon="pencil-alt"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M497.9 142.1l-46.1 46.1c-4.7 4.7-12.3 4.7-17 0l-111-111c-4.7-4.7-4.7-12.3 0-17l46.1-46.1c18.7-18.7 49.1-18.7 67.9 0l60.1 60.1c18.8 18.7 18.8 49.1 0 67.9zM284.2 99.8L21.6 362.4.4 483.9c-2.9 16.4 11.4 30.6 27.8 27.8l121.5-21.3 262.6-262.6c4.7-4.7 4.7-12.3 0-17l-111-111c-4.8-4.7-12.4-4.7-17.1 0zM124.1 339.9c-5.5-5.5-5.5-14.3 0-19.8l154-154c5.5-5.5 14.3-5.5 19.8 0s5.5 14.3 0 19.8l-154 154c-5.5 5.5-14.3 5.5-19.8 0zM88 424h48v36.3l-64.5 11.3-31.1-31.1L51.7 376H88v48z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </span>
    </button>
  </div>
</div>
`;
